base/src/com/google/idea/blaze/base/buildmodifier/BuildifierFormattingService.java

```diff
@@ -44,12 +48,20 @@ public final class BuildifierFormattingService extends AsyncDocumentFormattingSe
 
   static final Supplier<Optional<String>> binaryPath = Suppliers.memoize(() -> getBinary());
 
+  static ImmutableList<String> getCommandLineArgs(String binary, BuildFile buildFile) {
+    ImmutableList.Builder<String> cmd = ImmutableList.builder();
+    cmd.add(binary);
+    BlazeFileType type = buildFile.getBlazeFileType();
+    return cmd.add(fileTypeArg(type)).addAll(pathArg(buildFile)).build();
+  }
```

Comment:
nit: I think it will be better if we moved this method (and the private methods it uses) to `BuildFileFormatter.java` because it already contains other method like `getReplacements` which are used by classes in this package.

Reply:
Done, thanks for the suggestion 👍